README.md

```diff
@@ -195,6 +195,63 @@ this:
+and bottom of the element containing your scrollable content ([although this element
+  can be configured](#containing-elements-and-scrollableancestor)). When a waypoint is
+within these boundaries, it is considered to be "inside." When a waypoint passes
+beyond these boundaries, then it is "outside."
+
+The `onEnter` and `onLeave` props are fired as an element transitions from
+being inside to outside, or vice versa.
+
+The `topOffset` and `bottomOffset` properties can adjust the position of these
+boundaries. By default, the offset is `0px`. If you specify a positive value,
+then the boundaries will be pushed inward, toward the center of the page. If
+you specify a negative value for an offset, then the boundary will be pushed
+outward from the center of the page.
+
+Positive values of the offset props are useful when you have an element that
+overlays your scrollable area. For instance, if your app has a `50px` fixed
+header, then you may want to specify `topOffset: '50px'`, so that the
+`onEnter` callback is called when waypoints scroll into view from beneath the
+header.
+
+Negative values of the offset prop could be useful for lazy loading. Imagine if
+you had a lot of large images on a long page, and you didn't want to load them
+all at once. You can use React Waypoint to receive a callback whenever they're a
+certain distance from the bottom of the page. For instance, by specifying
+`bottomOffset: -200px`, then your `onEnter` callback would be called when
+the waypoint comes closer than `200px` from the bottom edge of the page.
+
+There are likely many more use cases for the offsets: be creative! Also, keep in
+mind that there are two boundaries, so there are two situations when
+the `onLeave` and `onEnter` callback will be called. By using the arguments
+passed to the callbacks, you can determine whether the waypoint has crossed
+the top boundary or the top boundary.
+
+## Containing elements and `scrollableAncestor`
+
+React Waypoint positions its [boundaries](#offsets-and-boundaries) based on the immediate parent of the
+Waypoint. This works great for situations where the waypoint is a direct
+descendant of the element that you're tracking the scrolling of.
+
+However, sometimes you may want to track the scrolling of a different
+ancestor element. For instance, if your scrollable element is a `ul`, then you
+will only want to place `li`s as its children. Waypoints render as a `span`,
+so you wouldn't want to use them as a direct descendant of the `ul`.
+
+Another situation where specifying a different scrollable ancestor would be
+if you want your waypoint to consider the scroll position of the entire page.
+The waypoint might be deeply nested in the DOM tree of the page.
+
+React Waypoint exposes the `scrollableAncestor` prop for these situations. Pass
+a node, and the Waypoint will use the scroll position of *that* node, rather
+than its immediate parent.
```

Comment:
Unless I'm mistaken, the "immediate parent" isn't correct. The waypoint will traverse up the DOM tree until it finds a scrollable container, and then it will use that as the `scrollableAncestor`.